src/itest/kotlin/nl/lifely/zac/itest/AppContainerTest.kt

```diff
@@ -22,5 +23,15 @@ class AppContainerTest : BehaviorSpec({
                 }
             }
         }
+        When("the metrics endpoint is called") {
```

Comment:
I think every test case normally starts with a `Given`. I think you may need to repeat the `Given` here as well insted of reusing that. Maybe that's the issue of the failing test?

Reply:
Normally it's also quicker to run the integration tests locally to check if they work instead of waiting for GitHub. ;-)